crypto-pgp/build.gradle.kts

```diff
@@ -12,4 +12,7 @@ dependencies {
   api(projects.cryptoCommon)
   implementation(libs.aps.gopenpgp)
   implementation(libs.dagger.hilt.core)
+  implementation(libs.thirdparty.kotlinResult)
+  implementation(libs.kotlin.coroutines.core)
+  implementation(libs.kotlin.coroutines.android)
```

Comment:
Is kotlin-coroutines-android actually needed?

Reply:
Not needed if we remove the default value of the `dispatcher` parameter.